carlos/rocker/index.html

```diff
@@ -0,0 +1,121 @@
+                        quis porttitor nisi, nec lacinia nunc. </p>
+                    <a href="entrada.html" class="btn btn-primario">Read more</a>
+                </div>
+            </article>
+            <article class="entrada-blog">
+                <div class="imagen">
+                    <img src="img/blog2.jpg" alt="Imagen blog">
+                </div>
+                <div class="contenido-blog">
+                    <h3 class="no-margin">A I</h3>
+                    <p>Quisque lorem ex, venenatis id mauris nec, pretium pharetra orci. Nam vehicula tristique imperdiet. Cras eu massa mi. Integer congue ullamcorper arcu, vehicula interdum est varius in. Sed eget elementum nisl, ut volutpat lorem. Nunc
+                        quis porttitor nisi, nec lacinia nunc. </p>
+                    <a href="entrada.html" class="btn btn-primario">Read more</a>
+                </div>
+            </article>
+            <article class="entrada-blog">
+                <div class="imagen">
+                    <img src="img/blog3.jpg" alt="Imagen blog">
+                </div>
+                <div class="contenido-blog">
+                    <h3 class="no-margin">Robotics</h3>
+                    <p>Quisque lorem ex, venenatis id mauris nec, pretium pharetra orci. Nam vehicula tristique imperdiet. Cras eu massa mi. Integer congue ullamcorper arcu, vehicula interdum est varius in. Sed eget elementum nisl, ut volutpat lorem. Nunc
+                        quis porttitor nisi, nec lacinia nunc. </p>
+                    <a href="entrada.html" class="btn btn-primario">Read more</a>
+                </div>
+            </article>
+        </main>
+        <aside class="cursos">
+            <h2>Stack</h2>
+            <ul class="cursos-lista">
+                <li class="curso">
+                    <i class="material-icons">adb</i>
+                    <span class="no-margin">HTML:</span>
+                    <span class="no-margin"><span>90%</span></span>
+
+                </li>
+                <li class="curso">
+                    <i class="material-icons">adb</i>
+                    <span class="no-margin">CSS: </span>
+                    <span class="no-margin">70%</span>
+
+                </li>
+                <li class="curso">
+                    <i class="material-icons">adb</i>
+                    <span class="no-margin">JavaScript: </span>
+                    <span class="no-margin">70%</span>
+
+                </li>
+                <li class="curso">
+                    <i class="material-icons">adb</i>
+                    <span class="no-margin">React: </span>
+                    <span class="no-margin">0%</span>
+
+                </li>
+            </ul>
+        </aside>
+    </div>
+
+    <footer class="site-footer">
+        <div class="contenedor">
```

Comment:
Same comment as in the header, you can set this class name to the footer element instead and delete this div